indico/modules/events/timetable/client/js/Timetable.module.scss

```diff
@@ -79,6 +81,17 @@
 
       &.selected {
         filter: opacity(0.9) brightness(0.95);
+        box-shadow: 4px 0 18px -2px $indico-blue;
+        margin-left: 4px;
+        z-index: 0;
+        border-color: $indico-blue !important;
+        outline: none !important;
+
+        &.parent-entry-selected {
+          box-shadow: none;
+          outline: none;
+          margin-left: 4px;
```

Comment:
```suggestion
          margin-left: 0.2em;
```

this is around half of the margin. looks better imo

![image](https://github.com/duartegalvao/indico/assets/27357203/83f54b5d-0619-4845-8bf8-7f0ee45eed4f)